src/Tizen.Multimedia.Util/ThumbnailExtractor/ThumbnailExtractor.cs

```diff
@@ -242,5 +235,103 @@ private static Native.ThumbnailExtractCallback GetCallback(TaskCompletionSource<
                 tcs.TrySetCanceled();
             });
         }
+
+        /// <summary>
+        /// Extracts the thumbnail for the given media with the specified path and size.
+        /// </summary>
+        /// <param name="path">The path of the media file to extract the thumbnail.</param>
+        /// <param name="size">The size of the thumbnail.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="path"/> is null.</exception>
+        /// <exception cref="FileNotFoundException"><paramref name="path"/> does not exist.</exception>
+        /// <exception cref="InvalidOperationException">An internal error occurs.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     The width or the height of <paramref name="size"/> is less than or equal to zero.
+        /// </exception>
+        /// <exception cref="FileFormatException">The specified file is not supported.</exception>
+        /// <returns>The result of extracting operation.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public static ThumbnailExtractionResult Extract(string path, Size size)
+        {
+            if (path == null)
+            {
+                throw new ArgumentNullException(nameof(path));
+            }
+
+            if (File.Exists(path) == false)
+            {
+                throw new FileNotFoundException("File does not exists.", path);
+            }
+
+            if (size.Width <= 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(size), size.Width,
+                    "The width must be greater than zero.");
+            }
+
+            if (size.Height <= 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(size), size.Height,
+                    "The height must be greater than zero.");
+            }
+
+            IntPtr thumbData = Marshal.AllocHGlobal(Marshal.SizeOf(typeof(IntPtr)));
```

Comment:
thumbData was not freed, 
but Why do you allocate memory for IntPtr 
Why not use `out IntPtr` ?

Reply:
I'll free it with `Marshal.FreeHGlobal(thumbData);`

Reply:
Right, it's not same case with https://github.com/Samsung/TizenFX/pull/919.
I'll change it.